bookShop/src/app/app.component.html

```diff
@@ -1,6 +1,9 @@
 <h1 #appTitle class="title"></h1>
 
-<app-book-component (buyEvent)="onBuy($event)"></app-book-component>
-<app-cart-component></app-cart-component>
+<nav>
```

Comment:
This nav should be wrapped in a header, I suppose.

Reply:
Fixed